corehq/apps/api/tests/user_resources.py

```diff
@@ -341,6 +341,50 @@ def test_update_fails(self):
             "non-editable field 'username', 'default_phone_number' must be a string\"}"
         )
 
+    def test_activate_user(self):
+        """Activate the user through the API"""
+        user = CommCareUser.create(domain=self.domain.name, username='inactive_user', password='*****',
+                                   created_by=None, created_via=None, is_active=False)
+        self.addCleanup(user.delete, self.domain.name, deleted_by=None)
+
+        activate_url = self.single_endpoint(user.get_id) + 'activate/'
+        response = self._assert_auth_post_resource(activate_url, json.dumps({}), content_type='application/json',
+                                                   method='POST')
+        updated_user = CommCareUser.get(user.get_id)
+        changes = UserHistory.objects.filter(user_id=user.get_id, changed_via=USER_CHANGE_VIA_API)
+
+        self.assertEqual(response.status_code, 202)
+        self.assertTrue(updated_user.is_active)
+        self.assertEqual(changes.count(), 1)
+
+    def test_deactivate_user(self):
```

Comment:
please split this test into 2, one for the normal user and one for the location user.